client/my-sites/upgrades/domain-management/list/item.jsx

```diff
@@ -9,29 +9,95 @@ import React from 'react';
 import CompactCard from 'components/card/compact';
 import DomainPrimaryFlag from 'my-sites/upgrades/domain-management/components/domain/primary-flag';
 import { type as domainTypes } from 'lib/domains/constants';
+import Spinner from 'components/spinner';
+import classNames from 'classnames';
 
 var Gridicon = require( 'components/gridicon' );
```

Comment:
This could be moved to the `import` section above (not introduced by this PR, but it stands out).